fintech-examples/fintech-example-tests/src/test/java/de/adorsys/fintech/tests/e2e/steps/FintechServer.java

```diff
@@ -0,0 +1,96 @@
+package de.adorsys.fintech.tests.e2e.steps;
+
+import com.tngtech.jgiven.annotation.BeforeStage;
+import com.tngtech.jgiven.integration.spring.JGivenStage;
+import de.adorsys.opba.db.domain.Approach;
+import de.adorsys.opba.db.repository.jpa.BankProfileJpaRepository;
+import de.adorsys.opba.protocol.xs2a.tests.e2e.sandbox.servers.SandboxServers;
+import io.restassured.RestAssured;
+import io.restassured.config.RedirectConfig;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.transaction.annotation.Transactional;
+
+@JGivenStage
+@Slf4j
+public class FintechServer<SELF extends FintechServer<SELF>> extends SandboxServers<SELF> {
+    private static final String ASPSP_PROFILE_BASE_URI = "http://localhost:20010";
```

Comment:
Instead of hardcoded value use 
```
@Autowired
private SmokeConfig config;

... somehere in code...
config.getAspspProfileServerUri(); // this is ASPSP_PROFILE_BASE_URI
```